internal/pkg/action/operator_uninstall.go

```diff
@@ -140,46 +136,54 @@ func (u *OperatorUninstall) deleteObjects(ctx context.Context, objs ...controlle
 	return waitForDeletion(ctx, u.config.Client, objs...)
 }
 
-func (u *OperatorUninstall) getInstallPlanResources(ctx context.Context, installPlanKey types.NamespacedName) (crds, csvs, others []controllerutil.Object, err error) {
-	installPlan := &v1alpha1.InstallPlan{}
-	if err := u.config.Client.Get(ctx, installPlanKey, installPlan); err != nil {
-		return nil, nil, nil, fmt.Errorf("get install plan: %v", err)
+func (u *OperatorUninstall) getInstallPlanResources(ctx context.Context, ipNamespace, installedCSV string) (crds, csvs, others []controllerutil.Object, err error) {
+	installPlanList := &v1alpha1.InstallPlanList{}
+	if err := u.config.Client.List(ctx, installPlanList, client.InNamespace(ipNamespace)); err != nil {
+		return nil, nil, nil, errors.New("error getting install plan resources")
```

Comment:
any reason not to just do `fmt.Errorf("error getting install plan resources")` to be consistent with the rest of the code.